includes/tracks/class-wc-site-tracking.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Nosara Tracks for Woo
+ */
+
+require_once dirname( __FILE__ ) . '/class-wc-tracks.php';
+require_once dirname( __FILE__ ) . '/class-wc-tracks-event.php';
+require_once dirname( __FILE__ ) . '/class-wc-tracks-client.php';
+
+/**
+ * This class adds actions to track usage of WooCommerce.
+ *
+ * @class   WC_Site_Tracking
+ * @package WooCommerce/Classes
+ */
+
+class WC_Site_Tracking {
+	/**
+	 * Send a Tracks event when a product is updated.
+	 *
+	 * @param int   $product_id Product id.
+	 * @param array $post WordPress post.
+	 */
+	public static function woocommerce_tracks_product_updated( $product_id, $post ) {
```

Comment:
Since this is in a class now you don't need to use woocommerce_ prefixes here which might make the class easier to read :) Not a blocker.